.github/workflows/ci.yml

```diff
@@ -9,22 +10,7 @@ on:
     branches: [ master ]
 
 jobs:
-  nose:
-    name: Nose tests
-    runs-on: ubuntu-20.04
-    steps:
-      - uses: actions/checkout@v2
-      - name: install deps for pycurl
-        run: sudo apt-get install -qq libssl-dev libcurl4-openssl-dev
-      - name: install pip dependencies
-        run: |
-          pip install .
-          pip install nose
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
-      - name: nosetests
-        run: nosetests
-  cube:
-    name: CUBE tests
+  test:
```

Comment:
I think we've settled on (but without fully agreeing) that we should keep unit tests outside `Dockerfile` and `docker build`. Would you revert that change?

p.s. multi-stage builds are nice but if we don't need it then we should not use it -- one of our Power9 machines on the MOC is running RHEL 7.6 with Docker version 1.13.1, which does not support multi-stage builds.

p.p.s. so to take advantage of fancy build features, images must be built on some other machine first (really, on my laptop using QEMU emulation) then exported to that machine. Cross-compilation in CI is something we can explore as future steps, but this is all very off-topic.

Reply:
I removed the test stage.

Reply:
Could you restore the nose tests back to `ci.yml` since it has been removed from `Dockerfile`?